docs/index.rst

```diff
@@ -104,14 +98,6 @@ Table of contents
 .. toctree::
    :maxdepth: 2
 
-   admin-docs
-   user-docs
+   installation
+   user-guide
    developer-docs
```

Comment:
Could we keep the changelog ?

Reply:
Yes, but I'm not sure whether it should be a top level heading or go under another section. @girder/developers  thoughts?

Reply:
I don't have a strong opinion about where, but I would like it to be included. How about as a top level heading so we don't have to argue about which other heading it should nest under?